feature-libs/asm/customer-360/components/asm-customer-promotion-listing/asm-customer-promotion-listing.component.html

```diff
@@ -45,13 +46,20 @@ <h4 class="cx-asm-customer-promotion-listing-heading-text">
         <ng-container *ngIf="entry.applied">
           <tr class="cx-asm-customer-promotion-listing-action">
             <td>
-              <cx-icon class="success" type="SUCCESS"></cx-icon>
+              <cx-icon
+                class="success"
+                type="SUCCESS"
+                *ngIf="!isCustomerCoupon"
```

Comment:
I see the `*ngIf="!isCustomerCoupon"` that has been used three times, How about only use one time? 
e.g:
```
 <ng-container *ngIf="!isCustomerCoupon">
              <td>
                <cx-icon class="success" type="SUCCESS"></cx-icon>
              </td>
              <td class="cx-asm-customer-promotion-listing-applied">
                {{ applied }}
              </td>
              <td
                *ngIf="showRemoveButton"
                class="cx-asm-customer-promotion-listing-action-separator"
              >
                |
              </td>
            </ng-container>
```

Reply:
agree, it will be more clear.